flocker/docs/bootstrap/_extension.py

```diff
@@ -0,0 +1,60 @@
+# Copyright Hybrid Logic Ltd.  See LICENSE file for details.
+
+"""
+Bootstrap extension entry point.
+"""
+
+from docutils import nodes
+from sphinx.writers.html import HTMLTranslator
+from . import _simple, _tabs
+
+
+def setup(app):
+    """
+    Entry point for sphinx extension.
+    """
+
+    for module in [_simple, _tabs]:
+        module.setup(app)
+
+
+class HTMLWriter(HTMLTranslator):
+    """
+    Overrides part of the default HTMLTranslator to provide specific
+    class names on some generated HTML elements.
+    """
+
+    def visit_table(self, node):
+        self._table_row_index = 0
+        self.context.append(self.compact_p)
+        self.compact_p = True
+        classes = ' '.join(['docutils', 'table', 'table-striped',
+                            self.settings.table_style]).strip()
```

Comment:
It seems like we could just override `table_style` instead of this function.

Alternatively, defining a node visitor (See `_tabs`, that adds the class to the node; `docutils.writers.html4css1.Writer.starttag` is where this is used).